From 70636ad110715b5e1ec6b08e24f0ddaf5df7186d Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Tue, 30 Jul 2024 19:00:03 +0100
Subject: [PATCH 1200/1215] ARM: dts: bcm2712: Fix invalid
 polling-delay-passive setting

This produces a hard fail on later (6.11) kernels.

See: https://lore.kernel.org/all/5802156.DvuYhMxLoT@rjwysocki.net/

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
@@ -40,7 +40,7 @@
 
 	thermal-zones {
 		cpu_thermal: cpu-thermal {
-			polling-delay-passive = <2000>;
+			polling-delay-passive = <1000>;
 			polling-delay = <1000>;
 			coefficients = <(-550) 450000>;
 			thermal-sensors = <&thermal>;
